config/routes.rb

```diff
@@ -69,6 +69,10 @@
     match 'pept2lca' => "api#pept2lca", via: [:get, :post]
     match 'taxa2lca' => 'api#taxa2lca', via: [:get, :post]
     match 'pept2prot' => 'api#pept2prot', via: [:get, :post]
+	  match 'pept2funct' => 'api#pept2funct', via: [:get, :post]
```

Comment:
tab/space

_[Original comment](https://github.ugent.be/unipept/unipept/pull/658#discussion_r16987) by @bmesuere on Thu Oct 11 2018 at 10:20._